app/controllers/services_controller.rb

```diff
@@ -95,11 +95,11 @@ def create
           end
         end
       else
-        flash[:error] =  service_route.capitalize + ' returned invalid data for the user id.'
+        flash[:alert] =  service_route.capitalize + ' returned invalid data for the user id.'
```

Comment:
Line is too long. [93/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.